pkg/apis/networking/v1alpha1/serverlessservice_lifecycle.go

```diff
@@ -45,6 +45,20 @@ func (sss *ServerlessServiceStatus) MarkEndpointsPopulated() {
 	serverlessServiceCondSet.Manage(sss).MarkTrue(ServerlessServiceConditionEndspointsPopulated)
 }
 
+// MarkEndpointsNotOwned marks that we don't own K8s service.
+func (sss *ServerlessServiceStatus) MarkEndpointsNotOwned(kind, name string) {
+	serverlessServiceCondSet.Manage(sss).MarkFalse(
+		ServerlessServiceConditionEndspointsPopulated, "NotOwned",
+		"Resource %s of type %s is not owned by SKS", name, kind)
+}
+
+// MarkEndpointsUnknown marks the ServerlessServiceStatus endpoints populated conditiohn to unknown.
+func (sss *ServerlessServiceStatus) MarkEndpointsUnknown(reason string) {
```

Comment:
`MarkBlahUnknown` doesn't scream `not ready` as the message will indicate, perhaps `MarkBlahNotReady`?